tests/js/spec/views/eventsV2/table/cellAction.spec.jsx

```diff
@@ -77,117 +74,86 @@ describe('Discover -> CellAction', function() {
 
   describe('per cell actions', function() {
     let wrapper;
+    let handleCellAction;
     beforeEach(function() {
-      wrapper = makeWrapper(view, initial);
+      handleCellAction = jest.fn();
+      wrapper = makeWrapper(view, handleCellAction);
       // Show button and menu.
       wrapper.find('Container').simulate('mouseEnter');
       wrapper.find('MenuButton').simulate('click');
-
-      browserHistory.push.mockReset();
     });
 
     it('add button appends condition', function() {
       wrapper.find('button[data-test-id="add-to-filter"]').simulate('click');
 
-      expect(browserHistory.push).toHaveBeenCalledWith({
-        pathname: '/organizations/org-slug/discover/results/',
-        query: expect.objectContaining({
-          query: 'event.type:transaction transaction:best-transaction',
-        }),
-      });
```

Comment:
Could we follow up this pull request to re-add test coverage that we're losing here around how each action is turned into querystring updates?

Reply:
Yea will do 👍